src/client/src/commons/form/borehole/chronostratigraphy/chronostratigraphyEditProfile.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState, useEffect } from "react";
+import {
+  Box,
+  Button,
+  Stack,
+  LinearProgress,
+  Table,
+  TableBody,
+  TableRow,
+  TableCell,
+  Typography,
+  IconButton,
+  ButtonGroup,
+} from "@mui/material";
+import LayerCard from "./layerCard";
+import LayerGap from "./layerGap";
+import {
+  useChronostratigraphies,
+  useChronostratigraphyMutations,
+} from "../../../../api/fetchApiV2";
+import { AddCircle, VisibilityOff, Visibility } from "@mui/icons-material";
+import { useTranslation } from "react-i18next";
+import NavigationChild from "./navigationChild";
+
+const getHeader = t => {
+  return [
+    { title: t("eon"), isVisible: true },
+    { title: t("era"), isVisible: true },
+    { title: t("period"), isVisible: true },
+    { title: t("epoch"), isVisible: true },
+    { title: t("subepoch"), isVisible: true },
+    { title: t("age"), isVisible: true },
+    { title: t("subage"), isVisible: true },
+  ];
+};
+
+const ChronostratigraphyEditProfile = ({
+  selectedStratigraphyID,
+  isEditable,
+  sx,
```

Comment:
Q: wird sx überhaupt vom Parent übergeben? Falls ja würde ich das eher vermeinden und wenn es nötig ist die styles vom Parent durchzureichen würde ich eher ein Objekt mit entsprechendem Namen machen das in der sx property einer Mui Komponente verwendet wird und das dann weitergeben.

Reply:
Ich möchte gewisse style properties z.B. `flex` im parent Objekt definieren. Als name für dieses Property habe ich _sx_ gewählt, weil es sich wie das _sx_ Property von Mui Konponenten verhalten soll. Dem Child ist egal welche styles der parent gerne hätte es verwendet diese einfach auf seine Mui root Komponente. Wie würdest du dieses Property denn nennen?

Das _sx_ Property von `ChronostratigraphyEditProfile` wird aktuell vom parent grad nicht gesetzt.

Reply:
Aber derzeit ist das soweit ich gesehen habe gar nicht variabel sondern wird einfach im parent hardcodiert reingegeben, dann sehe ich den Vorteil nicht so ;) 
```

 <NavigationChild
                sx={{ flex: "0 0 8em" }} ....
```
Falls es styles gibt, die nur sinnvoll sind im parent zu setzen würde ich es dann irgendwie FlexStyles oder was auch immer nennen. Aber ich kann auch gut damit leben dass es so bleibt :D

Reply:
Ich finde die Komponente ist so flexibler einsetzbar. Im Moment gibt es erst die Chronostratigraphie aber vor allem bei der LithologieViewProfile Komponente kann ich mir gut vorstellen, dass diese an einem anderen Ort wiederverwendet wird, an dem vielleich ein anderer Style benötigt wird. _FlexStyles_ passt mir nicht so, weil es nicht auf flex styles beschränkt ist. Der parent könnte auch andere styles setzen.